.github/workflows/test.yml

```diff
@@ -151,51 +151,39 @@ jobs:
   test-emscripten:
     name: Test Emscripten
     runs-on: ubuntu-latest
+    env:
+      PYODIDE_VERSION: '0.21.0-alpha.2'
+      PYTHON_VERSION: '3.10.2'
```

Comment:
After https://github.com/pyodide/pyodide/pull/2786 we should be able to calculate `PYTHON_VERSION` from `repodata.json` the same way as `EMSCRIPTEN_VERSION`.